src/app/pages/entities/entities.page.ts

```diff
@@ -0,0 +1,48 @@
+import { Component } from '@angular/core';
+import { NavController } from '@ionic/angular';
+
+@Component({
+  selector: 'app-entities',
+  templateUrl: 'entities.page.html',
+  styleUrls: ['entities.page.scss']
+})
+export class EntitiesPage {
+
+  entities: Array<any> = [
+    /* jhipster-needle-add-entity-page - JHipster will add entity pages here */
+  ];
+
+  constructor(public navController: NavController) { }
+
+  openPage(page) {
+    // Reset the content nav to have just this page
+    // we wouldn't want the back button to show in this scenario
+    this.navController.navigateForward(page.component);
```

Comment:
If this works for deep-linking like the code below used to, please remove the commented code below.

Reply:
I haven't implemented the entity generation part as I have to touch the generator project. I will do it next week I think